src/ocl/convolutionocl.cpp

```diff
@@ -188,6 +189,11 @@ std::vector<Solution> FindConvolution(const ExecutionContext& ctx,
                             force_attach_binary);
         });
     }
+    else
+    {
+        MIOPEN_LOG_W(
+            "Skip FindCore possible cause: user specified workspace is smaller than required");
+    }
```

Comment:
If this branch is taken the function will produce an empty result vector. Can we supply a default or naive result.